conductor/server/tasks/delete_device_deployments.json

```diff
@@ -0,0 +1,13 @@
+[
+    {
+        "name": "delete_device_deployments",
+        "retryCount": 5,
+        "timeoutSeconds": 10,
+        "inputKeys": [],
+        "outputKeys": [],
+        "timeoutPolicy": "RETRY",
+        "retryLogic": "EXPONENTIAL_BACKOFF",
+        "retryDelaySeconds": 60,
+        "responseTimeoutSeconds": 3600
```

Comment:
`if greater than 0, the task is rescheduled if not updated with a status after this time. Useful when the worker polls for the task but fails to complete due to errors/network failure.`

I understand it as - the task was scheduled, the worker didn't pick it up (or failed to report state, e.g. `IN_PROGRESS`), so we time it out after X seconds.

so I don't think it even applies here - we don't have our own workers, `HTTP_TASK` is a system task. btw. `3600` is a value from one of the examples. I think I'll set it to 0 or omit it altogether.

the difference vs `timeoutSeconds` - this timeout is applied only when the task is already in the `IN_PROGRESS` state. so this is what is intuitively understood as a timeout - we know the task is being performed, and we give it X seconds to complete.

https://netflix.github.io/conductor/metadata/#task-definition